docs/src/pages/components/data-grid/getting-started/getting-started.md

```diff
@@ -214,14 +221,44 @@ LicenseInfo.setLicenseKey(
 );
 ```
 
-The grid checks the key without making any network requests.
+### Security
+
+The MUI X checks the license key without making any network requests. It's air gappped.
```

Comment:
```suggestion
The data grid checks the license key without making any network requests.
```

Reply:
Once we have an isolated space for MUI X, we will move this content to cover the whole MUI X Pro/Premium plan, so I think that we can keep it generic.